packages/GA4Client/test/src/tests.js

```diff
@@ -200,6 +200,38 @@ describe('Google Analytics 4 Event', function () {
 
             done();
         });
+
+        it('should set the measurement ID as an Integration Attribute', function (done) {
+            var sandbox = sinon.createSandbox();
+            var mPStub = sinon.stub(window.mParticle);
+
+            // Kit init requires checking SDK version, which in turn requires
+            // the SDK to be initialized. The actual version number is not
+            // relevant to capturing measurment ID.
+            mPStub.getVersion.returns('X.XX.X');
+
+            window.mockGA4EventForwarder = new mockGA4EventForwarder();
+
+            mParticle.forwarder.init(kitSettings, reportService.cb, true);
+
+            // GTAG will fire an async get request so we are simply
+            // verifying that measurementID/clientID makes it into
+            // mParticle.setIntegrationAttribute via gtag
+            dataLayer[2][3]('testMeasurementId');
+
+            // Verify that data later triggers setClientId
+            mPStub.setIntegrationAttribute.calledWith(160, {
+                client_id: 'testMeasurementId',
+            });
+
+            // Set Integration Delay should be called twice upon init
+            // First, as true, then false after client ID is registered
+            mPStub._setIntegrationDelay.getCall(0).calledWith(160, true);
+            mPStub._setIntegrationDelay.getCall(1).calledWith(160, true);
```

Comment:
This is showing that it is being called with `true` both times, rather than `true` the first time then `false` the second time.